### PR TITLE
feat: import Hedera SDK JS, and build

### PR DESCRIPTION
**Pull request by bguiz**
## What

- Issue with '@hashgraph/cryptography' not allowing client-side usage of SDK lib
- Resolved by switching from bun to vue cli, as suggestion by @neurone

# Why

- Demonstrate how to build a front end application with the Hedera SDK, as it is not straightforward

## Refs

- Stackoverflow question: https://stackoverflow.com/q/78271342/194982

### CODE REVIEW

File: dapp-hedera-sdk/package.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-        "bundle": "bun build ./src/dapp.ts --outdir ./dist --format 'esm' --target 'browser'",
-        "serve": "npx http-server@14.1.1 ./dist -c-1 -p 8111"
-    },
-    "dependencies": {
-    }
+  "type": "module",
+  "scripts": {
+    "dev": "npm run build-vue && npm run static && npm run serve",
+    "static": "cp -r ./static/* ./dist/",
+    "build-bun": "bun build --outdir ./dist --format 'esm' --target 'browser' ./src/dapp.js",
```
Comment (michielmulders) [inline dapp-hedera-sdk/package.json:6]: Is there a need to keep the `build-bun` as it's not being used anymore or is this rather for backward compatibility?

**Review by michielmulders — APPROVED** [review]
Left one comment for you to consider, LGTM!

File: dapp-hedera-sdk/package.json (hunk 1 above)
Comment (bguiz) [reply dapp-hedera-sdk/package.json:6]: The build using bun is *currently* not working due to some issue that has a (long running) WIP fix. Ideally when that is merged, we can provide 2 ways to build the app. So I think of this as a marker/ reminder to come back to this later!

**Comment by bguiz** [comment]
merged manually in commit #1b93ad7f

**Pull request closed without merging**